docs/development/accessibility.md

```diff
@@ -776,76 +776,52 @@ Target group: developer
+1. Tastaturnutzbarkeit
+2. Fokusverfolgung
+3. HTML Elemente Überschriftenstruktur
+4. Kontraste
+5. Alt-Texte
+## Tabbing and Focus
+Please manually check your project for tabbing and focus quaility: 
 
+1.	All functionality must be operable by tabbing or number pad arrows. (2.1.1) 
+2.	Users can tab to any control, operate it, and tab away again. (2.1.2) 
+3.	Keyboard focus must be always visible. The element that has focus gets a corona. (2.4.7) 
+4.	Tabbing order must be sensible and predictable. The move of focus is following the meaning of the content and can be operated by tab. Moving focus should not jump around unpredictably. (2.4.3)
+5.	If an element gains focus it must not
+	* transfer users to a different context automatically and without warning i.e. open a new window. 
+	* Carry out an action automatically i.e. submit a form after a save-button received focus. 
+6.      Give users advanced warning when opening a new window i.e. by using the target attribute which provides an unambiguously machine-readable indication that a new window will open. (3.2.1)
+7.      Give users a submit-button to initiate change of context. 
 ## Structure 
-Check how your suggested element fits into our semantic structures. 
-1.	Write “well formed” HTML, employ HTML according to the specification with complete start and end tags and compliant nesting. Assistive technology will crash on missing start or end tags. (4.1.1)
-2.	We use aria landmarks and headlines to furnish ILIAS with semantic structures. Regions of the page and navigation are structured by aria landmarks. Content gets structured by headlines. These elements provide programmatically determinable semantic structures to convey information and relationships (1.3.1).  
-3.	Users must be able to skip ‘repeated content of second rank’ i.e. Secondary Panels (2.4.1). Use aria landmark role ‘complementary’ to make elements skippable. 
-4.	Make the DOM order matche the visual order (1.3.2). When the source order matches the visual order, everyone will read the content and interact with it in the same (correct) order. If this will not work use tabindex 0. 
-5.	Make sure that repeated navigational components occur in the same place and same relative order each time they appear. (3.2.3) 
-## Headings and Labels and Language
-6.	Our headings and labels have to be clear and descriptive (2.4.6). 
-7.	Provide clear and descriptive labels for any field that requires input. AND one of the following (3.3.2)
+Check how your project complies to our semantic structures: 
+
+8.	Write “well formed” HTML, employ HTML according to the specification with complete start and end tags and compliant nesting. Assistive technology will crash on missing start or end tags. (4.1.1). Use [validator](https://validator.w3.org/) to verfiy, [this test case](https://testrail.ilias.de/index.php?/tests/view/44245) may be helpful.  
```

Comment:
nur valides HTML, der restliche Text kann weg

Reply:
In case you keep it: `verfiy` → `verify`.